nss-cohort-36/keahua-arboretum-net-surfers
# Ts adding add plant

# Description

Plants can be added to biomes and should show up in report.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout TS-adding-add-plant`
4.  Create a biome, add a plant to biome, and see if it appears in the report.

# Checklist:
